src/components/LoadableComponentErrorBoundary/LoadableComponentErrorBoundaryPage.js

```diff
@@ -0,0 +1,61 @@
+import React from 'react';
+import { FormattedMessage } from '../../util/reactIntl';
+import { pathByRouteName } from '../../util/routes';
+import routeConfiguration from '../../routeConfiguration';
+import {
+  LayoutSingleColumn,
+  LayoutWrapperTopbar,
+  LayoutWrapperMain,
+  InlineTextButton,
+  Logo,
+} from '../../components';
+
+import css from './LoadableComponentErrorBoundary.module.css';
+
+export const LoadableComponentErrorBoundaryPage = () => {
+  const landingPagePath = pathByRouteName('LandingPage', routeConfiguration());
+  const handleOnClick = () => {
+    if (typeof window !== 'undefined') {
+      window.location = landingPagePath;
```

Comment:
We want the app to make full page refresh when navigating back LandingPage. I'm not sure if there is some other (better) way to trigger that or if this is ok.

Reply:
I think this is fine.